core/controllers/feedback_test.py

```diff
@@ -714,6 +717,7 @@ def test_get_recently_posted_feedback_messages(self):
 
         self.assertEqual(len(results), 2)
 
+        self.assertEqual(results[0]['author_id'], self.moderator_id)
```

Comment:
Why we check for ID when there should be a username? ditto elsewhere

Reply:
updated.